src/zimcheck/checks.cpp

```diff
@@ -362,11 +363,18 @@ void ArticleChecker::check(zim::Entry entry)
         return;
     }
 
-    const auto item = entry.getItem();
+    checkItem(entry.getItem());
+}
 
+void ArticleChecker::checkItem(const zim::Item& item)
```

Comment:
Rename it to `check_item_size` instead ?

Reply:
This function is responsible for all kind of checks on items' contents (`--empty`, `--url_internal`, `--url_external`, `--redundant`). Therefore the proposed name is not adequate. However, I changed the case from camelCase to snake_case for consistency with other methods of `ArticleChecker`.